plugins/inventory/linode.py

```diff
@@ -181,26 +181,6 @@ def _add_groups(self):
         for linode_group in self.linode_groups:
             self.inventory.add_group(linode_group)
 
-    def _filter_by_config(self, regions, types, tags):
```

Comment:
I'm not sure that it's a good idea to remove this function. I would keep it.

Reply:
Alright. I added the function back. Instead of pulling them out of the config again, though, I used the ones that were loaded as part of `_read_config_data`. Let me know what you think.

Reply:
That's totally fine! Thanks!